docs/overview.md

```diff
@@ -0,0 +1,44 @@
+# KILT Protocol
+
+KILT is a decentralized system to own and manage your identity. 
+It's a simple protocol for creating, claiming, issuing, presenting and verifying digital credentials attached to it. 
```

Comment:
I just didnt want to start the sentence with "KILT" again since the sentence right before also started with "KILT" so I feel its clear that "it" in the second sentence refers to the subject of the sentence before.